ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/compact-3node-cluster/provision/cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-provision-chain.yaml

```diff
@@ -0,0 +1,12 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-compact-3node-cluster-provision
+  steps:
+  - chain: ipi-conf-aws
+  - chain: ipi-install-aws
+  - ref: enable-qe-catalogsource
+  env:
+  - name: SIZE_VARIANT
+    default: 'compact'
+    documentation: "enable compact 3 node cluster on aws for CORS-2421"
```

Comment:
`SIZE_VARIANT ` parameter is not introduced by CORS-2421, you can describe what is `SIZE_VARIANT` used for? e.g. https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-ref.yaml#L13-L25

Reply:
OK, I have modified it.